packages/react-form/src/hooks/list/test/dynamiclist.test.tsx

```diff
@@ -203,6 +212,66 @@ describe('useDynamicList', () => {
         expect(wrapper).toContainReactComponent(TextField);
       });
     });
+
+    describe('dirty dynamic list', () => {
```

Comment:
So far there was no dirty related test

Reply:
hmm 🤔 @sylvhama . There are dirty tests in the baselist since that is where it is originally implemented

Reply:
So I dont think we will need to have it in the dynamiclist test

Reply:
same case for reset

Reply:
But `useList` test don't cover cases where we add or remove an item. Plus it does not hurt to make sure nothing break the default behavior.

Reply:
Oh I dont mean`useList`, I meant `useBaseList`. Because I noticed in this PR some tests like value changes are being re-created. In that case, could we just write the tests that cover the adding and removing?